submit.yml.erb

```diff
@@ -2,6 +2,8 @@
 batch_connect:
   template: "vnc"
 script:
+  job_environment:
+    ANSYS_MODULE: <%= version %>
```

Comment:
This commit won't have any effect in the future when I update the `pbs-ruby` library with a fix for https://github.com/OSC/pbs-ruby/issues/28. So we may have to come up with an alternative solution for the issue you are trying to resolve with this solution.

For example if there is no `template/before.sh` then we could do:

```yaml
# submit.yml.erb
---
batch_connect:
  template: "vnc"
  before_script: "# version = <%= version %>"
script:
  ...
```

This will then be injected in to the batch script.